editor.html

```diff
@@ -84,7 +84,6 @@ <h2><i class="fa fa-upload"></i> <strong>{{ load-title }}</strong></h2>
                 </form>
             </div>
```

Comment:
If we move `<input type="file" style="display: none" name="load-form-file-upload" id="file-upload-input">` inside the `<div id="load-alt">`, we can then remove all of `<div class="load-form">` as well.